README.rst

```diff
@@ -316,3 +316,24 @@ destroy the machine with::
 
 cornflow app  "http://vagrantfileIP:5000"
 airflow GUI  "http://vagrantfileIP:8080"
+
+Test conrflow
+~~~~~~~~~~~~~~~~~~
+
+To test conrflow first you will have to create a new database::
+
+    sudo su - postgres
```

Comment:
couldn't we do this inside the testing command? so the user only runs one comment in order to test everything?

Reply:
I could try to create an executable file that creates the database, runs the tests and then deletes the database, but said executable should have a linux version and a win version